pycode/memilio-simulation/memilio/simulation/bindings/models/abm.cpp

```diff
@@ -117,26 +134,6 @@ PYBIND11_MODULE(_simulation_abm, m)
         .def(py::self += mio::abm::TimeSpan{})
         .def(py::self - mio::abm::TimeSpan{})
         .def(py::self -= mio::abm::TimeSpan{});
-        
-    pymio::bind_class<mio::abm::TestParameters, pymio::EnablePickling::Never>(m, "TestParameters")
-        .def(py::init<double, double, mio::abm::TimeSpan, mio::abm::TestType>())
-        .def_readwrite("sensitivity", &mio::abm::TestParameters::sensitivity)
-        .def_readwrite("specificity", &mio::abm::TestParameters::specificity)
-        .def_readwrite("required_time", &mio::abm::TestParameters::required_time)
-        .def_readwrite("type", &mio::abm::TestParameters::type);
-
-    pymio::bind_CustomIndexArray<mio::UncertainValue<double>, mio::abm::VirusVariant, mio::AgeGroup>(
-        m, "_AgeParameterArray");
-    pymio::bind_CustomIndexArray<mio::abm::TestParameters, mio::abm::TestType>(m, "_TestData");
-    pymio::bind_Index<mio::abm::ExposureType>(m, "ExposureTypeIndex");
-    pymio::bind_ParameterSet<mio::abm::ParametersBase, pymio::EnablePickling::Never>(m, "ParametersBase");
-    pymio::bind_class<mio::abm::Parameters, pymio::EnablePickling::Never, mio::abm::ParametersBase>(m, "Parameters")
-        .def(py::init<int>())
-        .def("check_constraints", &mio::abm::Parameters::check_constraints);
-
-    pymio::bind_ParameterSet<mio::abm::LocalInfectionParameters, pymio::EnablePickling::Never>(
-        m, "LocalInfectionParameters")
-        .def(py::init<size_t>());
```

Comment:
Why was this removed again? I think we need it for the correct python bindings of the TestParameters, as the time and type are missing in the current bindings. Also, we wanted to include the serialization for the TestParameters once the auto-serialization has been done.

Reply:
Sorry, I made a mistake during the manual merging with the main branch. I completely forgot that you corrected the python bindings of the TestParameters in the issue as well. It's now added back.